packages/tocco-ui/src/EditableValue/example.js

```diff
@@ -65,6 +65,7 @@ class Example extends React.Component {
         datetime: '2017-01-25T15:15:00.000Z',
         decimal: 123456.78,
         duration: 3660000,
+        time: 0,
```

Comment:
why do you set 0 as example value and not a "real" value like 18:10

Reply:
i thought the time value is an object?

Reply:
That's right.